src/test/java/ai/elimu/LetterToAllophoneMappingsControllerTest.java

```diff
@@ -31,57 +30,54 @@
 import static org.hamcrest.CoreMatchers.*;
 import static org.junit.Assert.assertThat;
 
-@RunWith(MockitoJUnitRunner.class)
-public class LetterToAllophoneMappingsControllerTest {
-
-    @Mock
+public class LetterToAllophoneMappingsControllerTest extends RestControllerTest{
```

Comment:
The file being tested is `LetterToAllophoneMappingsRestController`, right? If so, this test should probably be renamed to `LetterToAllophoneMappingsRestControllerTest`.

Reply:
```suggestion
public class LetterToAllophoneMappingsControllerTest extends RestControllerTest {
```